Runtime/Scripts/LocationService.cs

```diff
@@ -72,9 +80,26 @@ public static IEnumerator InitalizeLocationService(int maxWait = 20, bool contin
       }
     }
 
-    public IEnumerator Start()
+    public static IEnumerator AssureLocation()
+    {
+         if (Input.location.status == LocationServiceStatus.Initializing)
+         {
+             yield return new WaitUntil(() => Input.location.status == LocationServiceStatus.Running);
```

Comment:
we should probably have a timeout for this also

Reply:
The timeout will be thrown in InitializeLocation Coroutine if it exceeds the 20 seconds limit